plans/tier1.fmf

```diff
@@ -370,3 +370,18 @@
   - name: reboot after conversion
     how: ansible
     playbook: tests/ansible_collections/roles/reboot/main.yml
+
+/detect_correct_boot_partition:
+   adjust:
+     enabled: true
+     when: trigger == commit
+     because: There are no UEFI images available on the Testing Farm yet.
```

Comment:
```suggestion
/detect_correct_boot_partition:
   adjust+:
     - enabled: false
       when: trigger == commit
       because: There are no UEFI images available on the Testing Farm yet.
```
We need to disable for the `commit` trigger